addons/overpressure/functions/fnc_overpressureDamage.sqf

```diff
@@ -33,6 +33,12 @@ private _var = if (isNil _varName) then {
 _var params ["_overpressureAngle","_overpressureRange","_overpressureDamage"];
 TRACE_3("cache",_overpressureAngle,_overpressureRange,_overpressureDamage);
 
+        
+// Range comes from scale of explosion, which is related to peak pressure and incident impulse.
+// Therefore it should directly scale maximum damage output. If player stands behind a booster, he deserves to be killed.
+// We will ignore the damage configs for now. The hardcoded threshold of 1 ensures potential lethality even for a weak backblast.
+_overpressureDamage = 1 + _overpressureRange * 0.2;
```

Comment:
This is not very proper way of doing that. Also I worry about respecting ACE overpressure damage scale setting.